modules/op-difference-between-non-versioned-and-versioned-cluster-tasks.adoc

```diff
@@ -0,0 +1,22 @@
+// This module is part of the following assembly:
+//
+// *cicd/pipelines/managing-nonversioned-and-versioned-cluster-tasks.adoc
+:_content-type: CONCEPT
+[id="difference-between-non-versioned-and-versioned-cluster-tasks_{context}"]
+= Difference between non-versioned and versioned cluster tasks
+
+.Difference between non-versioned and versioned cluster tasks
+[options="header"]
+|===
+
+| | Non-versioned cluster task | Versioned cluster task
```

Comment:
shall we add a third row to this table `intent` (stable pipelines vs uptodate pipelines) even though it is mentioned in detail in advantages vs dis-advantages table.

Reply:
We are strongly discouraged about repeating things. If I have to mention intent here, I will have to take out those points from the "advantages and disadvantages" section. I would prefer to retain them where they are currently.